README.md

```diff
@@ -52,18 +52,33 @@ go get -u github.com/juicedata/juicesync
 # Usage
 
 ```
-juicesync [options] SRC DST
-
-Options:
-  -end string
-    	the last keys to sync
-  -p int
-    	number of concurrent threads (default 50)
-  -q	change log level to ERROR
-  -start string
-    	the start of keys to sync
-  -v	turn on debug log
-  --help show the usage
+$ juicesync -h
+NAME:
+   juicesync - Usage: juicesync [options] SRC DST
+    SRC and DST should be [NAME://][ACCESS_KEY:SECRET_KEY@]BUCKET.ENDPOINT[/PREFIX]
+
+USAGE:
+   juicesync [global options] command [command options] [arguments...]
```

Comment:
we don't support sub-command here

Reply:
We do support the sub-command `help`

Reply:
it's actually generated by the module... but it's a good idea to make it's update automatically